lib/completions/endpoints/base.rb

```diff
@@ -226,6 +250,21 @@ def perform_completion!(dialect, user, model_params = {})
           end
         end
 
+        def normalize_function_ids!(function_buffer)
+          function_buffer
+            .css("invoke")
+            .each_with_index do |invoke, index|
+              if invoke.at("tool_id")
+                invoke.at("tool_id").content = "tool_#{index}" if invoke
```

Comment:
Isn't it better to have the function name here, or does the order matters? Just curious

Reply:
its does not really, it just needs to properly identify the specific call to the llm so the tool reply is clearly glued to it. going this way made the tests a bit easier to fix